src/app/loadout/SavedMods.tsx

```diff
@@ -22,19 +19,14 @@ interface Props {
 /**
  * Component for managing mods associated to a loadout.
  */
-function SavedMods({ defs, savedMods, onOpenModPicker, removeModByHash }: Props) {
+function SavedMods({ savedMods, onOpenModPicker, removeModByHash }: Props) {
   // Turn savedMods into an array of mod groups where each group is
   const groupedMods = useMemo(() => {
-    if (!defs.isDestiny2()) {
-      return [];
-    }
-
     const indexedMods = _.groupBy(savedMods, (mod) => mod.plug.plugCategoryHash);
-
     return Object.values(indexedMods).sort(sortModGroups);
-  }, [savedMods, defs]);
+  }, [savedMods]);
 
-  if (!defs.isDestiny2() || !Object.keys(savedMods).length) {
+  if (!savedMods.length) {
```

Comment:
@ryan-rushton i messed with the logic here but it seemed like some extraneous checks were happening?
`savedMods` by definition, seems like it would require the manifest to be loaded..

Reply:
Yeah the object.keys one is a little weird. From memory loadout can be opened with destiny 1 and destiny 2 gear and thus the corresponding manifest. That check was just an exit should we be in destiny 1 mode and it may have been there to satisfy types but I am not totally sure.